CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 * [Feature] Automatically connect if the `dsn_filename` (defaults to `~/.jupysql/connections.ini`) contains a `default` section
 * [Feature] Add `%sqlcmd connect` to see existing connections and create new ones (#632)
+* [Fix] Clearer error messages when failing to initialize a connection
```

Comment:
Do we need to add changelog entry for the new guide `db-drivers.md` ?

Reply:
https://github.com/ploomber/jupysql/pull/833/commits/138fab01769c9db7dca73a6affec8f72d02f6bee